templates/paragraphs/paragraph--oe-rich-text.html.twig

```diff
@@ -6,7 +6,9 @@
  * @see ./modules/contrib/paragraphs/templates/paragraph.html.twig
  */
 #}
-{% include '@oe-bcl/bcl-heading/heading.html.twig' with {
-  title: content.field_oe_title,
-} only %}
+{% if content.field_oe_title is not empty %}
```

Comment:
use the |field_value filter to avoid render meta giving false negatives

Reply:
done